### ISSUE TITLE
Clears the players inventory

### ISSUE THREAD

**Issue by Zoom4ikDan4ik** [CyberdyneCC/Thermos#579]
**Explanation of issue: Clears the players inventory when you exit the server or reconnect to the server. Tried to remove some plugins that could cause this bug, but nothing helps. Distributed on all servers.**

**Plugins Installed: http://image.prntscr.com/image/6be9bac3d0634b6e8a98fe1907b2256f.png**

**Thermos Version: latest build 58 alpha**

**Forge Version: 1.7.10-1614**

**Comment by Time6628** [comment]
Remove plugins one by one until you find the one that is causing it.

**Comment by braunly** [comment]
Does your host have enough disk space?

**Comment by Zoom4ikDan4ik** [comment]
Yes, the disk has about 100 GB of memory to the same folder servers have been given permission 777. Some holders of kernel Thermos the same problem.

**Comment by Time6628** [comment]
Try build 57.

**Comment by Zoom4ikDan4ik** [comment]
Installed build 57, but it has not produced results.
